server/.env.example

```
@@ -1,18 +1,9 @@
 
-TYPEORM_CONNECTION=mysql
-TYPEORM_HOST=127.0.0.1
-TYPEORM_PORT=3306
-TYPEORM_DATABASE=nlw_happy
-TYPEORM_USERNAME=root
-TYPEORM_PASSWORD=password
-TYPEORM_LOGGING=true
-TYPEORM_ENTITIES=src/models/*.ts
-TYPEORM_MIGRATIONS=src/database/migrations/*.ts
-TYPEORM_MIGRATIONS_DIR=src/database/migrations
+DATABASE_URL="file:./database/db.sqlite"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)